docs/tutorials/getting_started.ipynb

```diff
@@ -72,7 +72,7 @@
     "\n",
     "from torch.utils.data import DataLoader\n",
     "\n",
-    "from torchgeo.datasets import NAIP, ChesapeakeDE, stack_samples\n",
```

Comment:
`stack_samples` is used in the `DataLoader` initialization below.

Reply:
Sorry, missed this, revert this commit.